app/frontend/src/features/communities/events/EventsList.tsx

```diff
@@ -50,6 +56,13 @@ export default function EventsList({ community }: EventsListProps) {
   return (
     <>
       <SectionTitle icon={<CalendarIcon />}>{EVENTS_TITLE}</SectionTitle>
+      <Button
+        component={Link}
+        className={classes.createEventButton}
+        to={newEventRoute}
+      >
```

Comment:
Can this be the same styling (should probably be shared) as the "Create discussion" button on the discussions listing page